lib/pages/send_page.dart

```diff
@@ -149,12 +149,13 @@ class _SendPageState extends State<SendPage> {
   }
 
   Future<SendStatus> _validate(String addr, TokenWithAmount token, String amt,
-      [bool skipAsync = false]) async {
+      [bool skipAsync = false,bool forceFetch=false]) async {
     var isValidAddr = await _isValidAddress(addr);
     var balance = getSelectedTokenBalance(token);
     var selectedAccount = ReefAppState.instance.model.accounts.accountsList.singleWhere((e) =>
             e.address == ReefAppState.instance.model.accounts.selectedAddress);
     var hasEnoughForEvmTx = hasBalanceForEvmTx(selectedAccount);
+    if(forceFetch)selectedAccount.isEvmClaimed=true;
```

Comment:
we shouldn't set values here on selectedAccount - the selectedAccount should already be updated when event comes from indexer - we're waiting for indexer to emit event in the bind flow so that should update selectedAccount in util-lib. Need to figure out if that is updated and if/why don't we get updated selectedAccount here